ckan/lib/cli.py

```diff
@@ -625,6 +628,35 @@ def clear(self):
         else:
             clear(package_id)
 
+
+    def remove_orphans(self):
+        indexed_package_ids = []
+        rows = 100
+        while True:
+            search = logic.get_action('package_search')({},{
+                                'q': '*:*',
+                                'start': rows - 100,
+                                'rows': rows,
```

Comment:
this will fetch 100 rows, then 200, then 300...

Reply:
@wardi ah, I thought start and rows were required. Okay that silly while loop has been removed, so it will now just get the entire index.